readme.md

```diff
@@ -18,13 +20,23 @@ const isAbsoluteUrl = require('is-absolute-url');
 isAbsoluteUrl('https://sindresorhus.com/foo/bar');
 //=> true
 
+isAbsoluteUrl('http://sindresorhus.com/foo/bar');
+//=> true
+
+isAbsoluteUrl('ftp://sindresorhus.com/foo/bar');
+//=> false
+
+isAbsoluteUrl('ftp://sindresorhus.com/foo/bar', { httpOnly: false });
+//=> true
+
 isAbsoluteUrl('//sindresorhus.com');
 //=> false
 
 isAbsoluteUrl('foo/bar');
 //=> false
 ```
 
+By default only `http` and `https` links are validated.  If you wish to check absolute URLs of protocols other than `http` (for example, `ftp:`, `mailto:`), use `isAbsoluteUrl(mayBeURL, { httpOnly: false })`.
```

Comment:
These are not *links*.

Reply:
The option should have proper API docs like done here: https://github.com/sindresorhus/boxen#api